### PR TITLE
Fix traceback error in IOS and IOSXR when ran without config

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Fix traceback error in IOS and IOSXR when ran without config, which fixes the issue raised in #61321

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ios and iosxr
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by ganeshrn — APPROVED** [review]

**Review by danielmellado — APPROVED** [review]
Seems quite straightforward, while it LGTM, let's wait for Zuul to provide feedback before merging. Thanks!

**Pull request merged**
